### DISCUSSION TITLE
New Orders for the AI Crew

### DISCUSSION THREAD

**Discussion by ShrebTV** [FakeFishGames/Barotrauma#8152] [Ideas and suggestions]
In my singleplayer save I've noticed a few things which are usually annoying to deal with.
- Security AI attacks when you hit the crew with something minor accidentally
- Medic injected me with 4 anaparalyzant at once for my paralysis before i could swap to them to stop it
- Crafting takes very long when youre alone
- AI Sorting seems to be random

Ive thought about some of this and want to propose following solutions.
- Make a "Do not attack Crewmembers" order for security bots, Huskified Crewmembers are of course an exception to this.
- make an order where you can tell medic bots to only heal others when they are under a certain percentage of health, Make medic 
   bots actually wait for the effect of drugs they use so they dont use 4 anaparalyzant where 1 would be plenty.
- A Crafting order where you can specify which item a certain bot should craft at a certain fabricator until they have a specified 
   number of the item they are crafting. This would of course have to take into account double crafts like the security talent where you 
   can get double output from an ammo craft. Possibly even have the option to specify a container to put the crafts into.
- A simple order where you can tell bots where to put which items. Having this while already inside a campaign would help a lot with 
   organization and lag since the bots cleanup items order would be more effective. Having even having groups of items would be great. Something along the lines of "This is the ores container, bots will gather ores from around the sub and put them in this container when there is space" would be great. I do get that organization is a big part in the game but it is like 25-50% of the playtime in singleplayer, which is a bit much at times.

**Comment by thegrb93** [comment]
An "Operate Equipment: Fabricator" would be amazing. Simply select whatever item in the fabricator to make and the bot will handle it. The one thing making single-player tedious at the moment for me is the fabricating.

**Reply by kitpaddel** [reply]
There is a mod to operate fabricators by bots:

[https://steamcommunity.com/sharedfiles/filedetails/?id=2592748017](https://steamcommunity.com/sharedfiles/filedetails/?id=2592748017)

**Reply by thegrb93** [reply]
Nice, thanks
